### PR TITLE
Phase 2 of ORM Reload project

### PR DESCRIPTION
**Pull request by kuanf**
Add advisor

### CODE REVIEW

File: registry/registry.go [hunk 1]
```diff
@@ -19,24 +19,35 @@ package registry
...
-type ORMRegistry struct {
-	// registry is defined to find orm and mappings by Owned Object
-	registry map[corev1.ObjectReference]ORMEntry
+type ResourceMappingRegistry struct {
 	// ownerRegistry is defined to find orm and mappings by Owner Object
-	ownerRegistry map[corev1.ObjectReference]ORMEntry
+	// legend: [owner objref][orm key][target objref][owner path]owned path
+	ownerRegistry map[corev1.ObjectReference]ResourceMappingEntry
+	// ownedRegistry is defined to find orm and mappings by owned resource
+	// legend: [owner objref][orm key][owner objref][owned path]owner path
```
Comment (pallavidn) [inline registry/registry.go:42]: legend for ownedRegistry ->
// legend: [owned objref][orm key][owner objref][owned path]owner path

File: registry/registry.go [hunk 2]
```diff
@@ -19,24 +19,35 @@ package registry
...
+// ResourceMappingEntry is defined for registry to search orm and all registered mappings
+type ResourceMappingEntry map[types.NamespacedName]ObjectEntry
 
-type ORMRegistry struct {
-	// registry is defined to find orm and mappings by Owned Object
-	registry map[corev1.ObjectReference]ORMEntry
+type ResourceMappingRegistry struct {
 	// ownerRegistry is defined to find orm and mappings by Owner Object
-	ownerRegistry map[corev1.ObjectReference]ORMEntry
+	// legend: [owner objref][orm key][target objref][owner path]owned path
```
Comment (pallavidn) [inline registry/registry.go:39]: change to target -> owned as below -
// legend: [owner objref][orm key][owned objref][owner path]owned path

**Review by pallavidn — APPROVED** [review]
LGTM

File: registry/registry.go (hunk 1 above)
Comment (kuanf) [reply registry/registry.go]: updated. thanks

File: registry/registry.go (hunk 2 above)
Comment (kuanf) [reply registry/registry.go]: updated, thanks

**Pull request merged**